packages/atomic/src/index.html

```diff
@@ -108,7 +108,11 @@
             <atomic-numeric-range start="40" end="50" end-inclusive="true"></atomic-numeric-range>
           </atomic-numeric-facet>
           <atomic-date-facet field="date" label="Date"></atomic-date-facet>
-          <atomic-category-facet field="geographicalhierarchy" label="World Atlas"></atomic-category-facet>
+          <atomic-category-facet
+            field="geographicalhierarchy"
+            label="World Atlas"
+            enable-facet-search
```

Comment:
makes it easier to quickly test the feature, we should revisit in the future if it should be enabled by default